node/src/chain/acceptor.rs

```diff
@@ -55,7 +55,7 @@ pub(crate) enum RevertTarget {
 /// attestation and transactions full verifications.
 /// Acceptor also manages the initialization and lifespan of Consensus task.
 pub(crate) struct Acceptor<N: Network, DB: database::DB, VM: vm::VMExecution> {
-    /// the tip
+    /// the Tip
```

Comment:
I actually meant to capitalize the t in `The`, since it's the beginning of the line :-)

Reply:
Done